content/es/docs/concepts/security/overview.md

```diff
@@ -0,0 +1,152 @@
+
+Provedor IaaS        | Link |
+-------------------- | ------------ |
+Alibaba Cloud | https://www.alibabacloud.com/trust-center |
+Amazon Web Services | https://aws.amazon.com/security/ |
+Google Cloud Platform | https://cloud.google.com/security/ |
+IBM Cloud | https://www.ibm.com/cloud/security |
+Microsoft Azure | https://docs.microsoft.com/en-us/azure/security/azure-security |
+VMWare VSphere | https://www.vmware.com/security/hardening-guides.html |
+
+{{< /table >}}
+
+### Seguridad de la Infraestructura {#infrastructure-security}
+
+Sugerencias para proteger su infraestructura en un clúster de Kubernetes:
+
+{{< table caption="Infrastructure security" >}}
+
+Área de Interés para la Infraestructura de Kubernetes | Recomendación |
+--------------------------------------------- | -------------- |
+Acceso de red al servidor API (Plano de Control) | Todo acceso público al plano de control del Kubernetes en la Internet no está permitido y es controlado por listas de control de acceso a la red restrictas a un conjunto de direcciones IP necesarios para administrar el clúster.|
+Acceso a la red de los Nodos | Los nodos deben ser configurados para _sólo_ aceptar conexiones (por medio de listas de control de acceso a la red) desde el plano de control en los puertos especificados y aceptar conexiones para servicios en Kubernetes del tipo NodePort y LoadBalancer. Si es posible, estos nodos no deben exponerse públicamente en la Internet.
+Acceso de la API del Kubernetes al proveedor de la Cloud | Cada proveedor de la nube debe dar un conjunto de permisos al plano de control y nodos del Kubernetes. Es mejor otorgar al clúster el permiso de acceso al proveedor de nube siguiendo el [principio del mínimo privilegio](https://en.wikipedia.org/wiki/Principle_of_least_privilege) para los recursos que necesite administrar. La [documentación del Kops](https://github.com/kubernetes/kops/blob/master/docs/iam_roles.md#iam-roles) fornece informações sobre as políticas e roles do IAM.
+Acceso al etcd | El acceso al etcd (banco de dados do Kubernetes) debe ser limitado apenas al plano de control. Dependiendo de su configuración, debería intentar usar etcd sobre TLS. Mayores informaciones pueden ser encontradas en la [documentación del etcd](https://github.com/etcd-io/etcd/tree/master/Documentation).
+Encriptación etcd | Siempre que sea posible, es una buena práctica encriptar todas las unidades de almacenamiento, el etcd mantiene el estado de todo el clúster (incluidos los Secretos), su disco debe estar encriptado.
+
+{{< /table >}}
+
+## Cluster
+
+Existe dos áreas de preocupación para proteger o Kubernetes:
+
+* Protección de las configuraciones de los componentes del clúster.
+* Protección de las aplicaciones que se ejecutan en el clúster.
+
+### Componentes del Clúster {#cluster-components}
+
+Si desea proteger su clúster de accesos accidentales o maliciosos y adoptar
+buenas prácticas de seguridad, a continuación los consejos sobre
```

Comment:
```suggestion
buenas prácticas de seguridad, a continuación ofrecen consejos sobre
```

Reply:
@electrocucaracha thanks! could be "a continuación sigue estos consejos sobre" ?